src/runtime/start/start-module.ts

```diff
@@ -144,6 +141,12 @@ export function createStartModuleContent(config: StartModuleConfig): string {
     }  
   `
 
+  if (config.disableServer) {
+    content += stripIndent`
+      process.env.NEXUS_DISABLE_SERVER = 'false'
```

Comment:
I have an idea for this which we can discuss later. Basically the problem is we have no way to force/do lazy boot time logic. For now this is a great solution!